samples/app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gandi.translate-sample",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
```

Comment:
Following master branch, you have to declare `babel-cli` else you won't be able to launch extract_messages